apps/fluent-tester/src/TestComponents/Divider/DividerTest.tsx

```diff
@@ -100,17 +102,36 @@ const dividerSections: TestSection[] = [
   },
 ];
 
+const mobileDividerSections: TestSection[] = [
+  {
+    name: 'Mobile Dividers',
+    testID: DIVIDER_TESTPAGE,
+    component: MobileDividers,
+  },
+  {
+    name: 'Customised Mobile Dividers',
+    testID: DIVIDER_TESTPAGE,
+    component: CustomisedMobileDividers,
+  },
+];
+
 export const DividerTest: React.FunctionComponent = () => {
   const status: PlatformStatus = {
     win32Status: 'Experimental',
     uwpStatus: 'Backlog',
-    iosStatus: 'Backlog',
+    iosStatus: 'Experimental',
     macosStatus: 'Backlog',
-    androidStatus: 'Backlog',
+    androidStatus: 'Experimental',
   };
 
-  const description =
-    'A Divider is a visual separator that can contain content (text or an icon). Dividers can be horizontal or vertical. As of now, the component is being designed for win32 only.';
+  const description = 'A Divider is a visual separator that can contain content (text or an icon). Dividers can be horizontal or vertical';
```

Comment:
Maybe update this to remove the note of text and icon for mobile?

Reply:
My understanding was different , thanks for the clarification.